src/api/db.ts

```diff
@@ -12,7 +12,7 @@ const init = (password: string) => {
   try {
     store = new Store({ encryptionKey: password, clearInvalidConfig: true });
     // Write something to the store to actually create the file
-    store.set('accounts', []);
+    store.set('accounts', {});
```

Comment:
Wouldn't this override the accounts every time `init` is called? Maybe check if `accounts` exists before writing?

Reply:
`init` is only for initializing the store and will delete anything in the store anyway!